eslint.config.js

```diff
@@ -22,7 +22,41 @@ export default antfu({
     rules: baseRules,
 }, {
     files: ['**/*.ts', '**/*.tsx'],
-    ignores: ['**/*.d.ts', '**/vite.config.ts', 'playwright.config.ts'],
+    ignores: ['**/*.d.ts', '**/vite.config.ts', 'playwright.config.ts', '**/*.spec.ts', '**/*.spec.tsx`'],
+    plugins: {
+        header,
+    },
+    rules: {
+        'header/header': [
+            2,
+            'block',
+            [
+                '*',
+                ' * Copyright 2023-present DreamNum Inc.',
+                ' *',
+                ' * Licensed under the Apache License, Version 2.0 (the "License");',
+                ' * you may not use this file except in compliance with the License.',
+                ' * You may obtain a copy of the License at',
+                ' *',
+                ' *     http://www.apache.org/licenses/LICENSE-2.0',
+                ' *',
+                ' * Unless required by applicable law or agreed to in writing, software',
+                ' * distributed under the License is distributed on an "AS IS" BASIS,',
+                ' * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.',
+                ' * See the License for the specific language governing permissions and',
+                ' * limitations under the License.',
+                ' ',
+            ],
+            2,
+        ],
+
+        // code readability
+        complexity: ['error', { max: 20 }],
```

Comment:
The new changes can have a significant impact on the codebase. 🤯

Reply:
Yes. Let's make it a warning for now.